src/library.tsx

```diff
@@ -15,7 +15,7 @@ const bundleName = `main-${packageJson.version}`;
 
 const scriptDomain =
   (() => {
-    if (typeof window === 'undefined') return '';
+    if (typeof window === 'undefined' || typeof document === 'undefined') return '';
```

Comment:
do we want to have the guard on `loadJupiter` function too

Reply:
I think it's fine, it's only called when previous checks are passed